app/SignupUseCaseFactory.java

```diff
@@ -14,13 +15,13 @@
 public class SignupUseCaseFactory {
     private SignupUseCaseFactory() {}
 
-    public static SignupView create(ViewManagerModel viewManagerModel, LoginViewModel loginViewModel, SignupViewModel signupViewModel, SignupUserDataAccessInterface accessObject) {
-        SignupController signupController = createSignupUseCase(viewManagerModel, loginViewModel, signupViewModel, accessObject);
+    public static SignupView create(ViewManagerModel viewManagerModel, LoginViewModel loginViewModel, SignupViewModel signupViewModel, SignupUserDataAccessInterface accessObject, LoggedInViewModel loggedInViewModel) {
```

Comment:
LoggedInViewModel should be passed to LoginUseCaseFactory and not SignupUseCaseFactory.